templates/common/infra/terraform/applicationinsights_variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "location" {
+  description = "The supported azure location where the resource deployed"
+  type        = string
+}
+
+variable "rg_name" {

```

Comment:
Is snakecase tf standard?

Reply:
it is a common convention, https://www.terraform-best-practices.com/naming